cartographer/mapping/map_builder.cc

```diff
@@ -230,6 +239,7 @@ void MapBuilder::LoadMap(io::ProtoStreamReader* const reader) {
     }
   }
   CHECK(reader->eof());
+  sparse_pose_graph_->AddConstraintsFromProto(pose_graph);
```

Comment:
Pass along only the repeated constraints? The other parts of the `pose_graph` are not used by `AddConstraintsFromProto()`, right?

Reply:
Not yet, but they will. `SPG::AddConstraintsFromProto` gets renamed to `AddDataFromProto` in the next PR (#548), and loading IMU data, odometry and trajectory connectivity will also be added to this function in #548, #550 and #552.

Reply:
Ok, let's refactor it then. Let's pass only the constraints here.

Reply:
So, you want me to pass an
`::google::protobuf::RepeatedPtrField< ::cartographer::mapping::proto::SparsePoseGraph_Constraint >` here?

Reply:
How do you want me to pass only the constraints? Pass a `::google::protobuf::RepeatedPtrField< ::cartographer::mapping::proto::SparsePoseGraph_Constraint >`?